service-api/src/main/java/greencity/dto/bag/EditAmountOfBagDto.java

```diff
@@ -13,9 +13,9 @@
 public class EditAmountOfBagDto {
     @NotNull
     @Min(1)
-    private Long minAmountOfBigBags;
+    private Long minQuantity;
```

Comment:
Same here
This DTO is used to edit `tariffLimist`
`@PatchMapping("/setLimitsByAmountOfBags/{tariffId}")`
Have you checked that it works fine?